### PR TITLE
Add table and mock data to reports view

### LINKED ISSUE #3: Table Display of Heroku data - Integration + UI Build
- [x] In the Main view component, build a simple table from the Vuetify library that displays the json data from mock data. 

- [x] Create columns on the table for the following fields: 
( Date, Shouter Slack Handle, Shoutee Slack Handle, Message, Company Values );

```[{"channel_name":"dev-slack-testing2","date":"Fri Oct 16 2020","shoutee":{"email":"elliot.mackinnon@techtonic.com","first_name":"elliot","last_name":"mackinnon","slack_handle":"Elliot Mackinnon","user_id":"U01AD22LWCX"},"shouter":{"email":"travis.gee@techtonic.com","first_name":"travis","last_name":"gee","slack_handle":"Travis","user_id":"U01B6CX7L9X"},"shoutout":"Hello","values":"Company value number 1"}]```

### PR DESCRIPTION
**Pull request by broxburyTechtonic**
### Type of change:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling - no new features

### Why is this change required? What problem(s) does it solve?
This change implements mock data and a simple vuetify table into the reports view. 

### Were there any challenges while implementing this feature? If so, how were they addressed?
Figuring out how to update and or modify data before being inserted into the rows. I am going to watch a slots video tomorrow to do some further digging on how that works. 
I also think we should think about making a table component to make things more dynamic. 
### Link issues
Closes #3

### CODE REVIEW

File: client/src/Views/Reports.vue [hunk 1]
```diff
@@ -1,10 +1,40 @@
...
   <h2>Reports</h2>
+  <v-data-table
+    :headers="tableHeaders"
+    :items="recentShouts"
+    item-key="shoutId"
+    class="elevation-1"
+    >
+  </v-data-table>
+
+</div>
```
Comment (tgeeTechtonic) [inline client/src/Views/Reports.vue:12]: This should probably be an article and not a div

File: client/src/shared/mockData.js [hunk 2]
```diff
@@ -1,22 +1,65 @@
...
+             first_name: "travis",
+             last_name: "gee",
+             slack_handle: "Travis",
+             user_id: "U01B6CX7L9X",
+           },
+           shoutout: "Hi there",
+           values: "Company value number 1",
+           shoutId: 3,
+         },
+       ];
```
Comment (tgeeTechtonic) [inline client/src/shared/mockData.js:65]: We should also vary the data a little so that it is apparent when the table is sorting

**Review by tgeeTechtonic — APPROVED** [review]
Looks good!

File: client/src/shared/mockData.js (hunk 2 above)
Comment (broxburyTechtonic) [reply client/src/shared/mockData.js:65]: Good call, I tested it a little bit, but I can make a ticket to change the data.

Comment (tgeeTechtonic) [reply client/src/shared/mockData.js:65]: We can seed the db as well, so it's up to you if you want to wait or alter the current mock data now

**Review by emackinnonTechtonic — APPROVED** [review]

**Pull request merged**